src/hooks/useMedia.tsx

```diff
@@ -0,0 +1,15 @@
+import { useState, useLayoutEffect } from 'react';
+
+export const useMediaQuery = (query: string): boolean | undefined => {
+  const [matches, setMatches] = useState<boolean>();
+
+  useLayoutEffect(() => {
+    const mq = window.matchMedia(query);
```

Comment:
is this short for media query?

Reply:
Mq is media query I guess. Either one really :)